src/storage/worditem-indexeddb-driver.js

```diff
@@ -437,4 +437,43 @@ static get currentDate () {
     }
     return wordItem
   }
+
+  getByStorageID (dataItems, ID) {
+    return dataItems.find(item => this.makeIDCompareWithRemote(item) === ID)
+  }
+
+  comparePartly (changeItem, sourceItem) {
```

Comment:
please add method/param description comments

Reply:
I also wonder if maybe diffAndMerge is a better name for this method? 'compare' implies that it just returns a comparison, but it actually modifies the object that is passed in arguments and returns the modified object, right?

Reply:
This method was removed